src/javascripts/legacy_app.js

```diff
@@ -41,6 +42,12 @@ var App = {
 
   logClosedModal: function() {
     console.log('Closed a modal.');
+  },
+
+  storeClosedAt: function() {
+    var closedAt = new Date();
+    console.log('CLOSING', closedAt);
+    this.store('lastClosedAt', closedAt);
```

Comment:
updated, but:
- the single existing test was failing
- this repo has no CI
- some examples in the code and some examples in tests means nobody knows where to look